src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -229,4 +278,18 @@ public void registerRecipeSerialziers(RegistryEvent.Register<IRecipeSerializer<?
         event.getRegistry().register(new ConditionalRecipe.Serializer<IRecipe<?>>().setRegistryName(new ResourceLocation("forge", "conditional")));
 
     }
+
+    @EventBusSubscriber(value = Dist.CLIENT, modid = "forge", bus = Mod.EventBusSubscriber.Bus.MOD)
+    public static class ClientEvents
+    {
+        @SubscribeEvent
+        public static void registerFluidTextures(TextureStitchEvent.Pre event)
+        {
+            if (enableMilkFluid && PlayerContainer.LOCATION_BLOCKS_TEXTURE.equals(event.getMap().getTextureLocation()))
+            {
+                event.addSprite(STILL_MILK_TEXTURE);
+                event.addSprite(FLOWING_MILK_TEXTURE);
+            }
+        }
+    }
```

Comment:
Why is this needed? `FluidModel#getTextures` calls `ForgeHooksClient#getFluidMaterials` which gets the textures from the FluidAttributes.

Reply:
Is that new for 1.13+? (since the addition of vanilla fluids probably) I recall a discussion with KingLemming that this was required, and did not hear of it being resolved since then.

Reply:
The fluid system got rewritten in 1.14+ (1.13 didn't have one at all), and if memory serves giga made it so it auto adds the textures defined in FluidAttributes to the correct atlas so that modders don't have to have a bunch of boiler plate doing so.

Reply:
Removed the event listener